udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step2AddDatasets.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <Container>
+    <Stepper :steps="steps" :currentStep="1"/>
+    <Well
+      color="blue-cumulus"
+      weight="regular"
+      class="fr-mb-2w"
+    >
+      <div class="fr-grid-row">
+        <div class="fr-col-auto fr-mr-3v">
+          <span class="fr-icon-info-line" aria-hidden="true"></span>
+        </div>
+        <div class="fr-col">
+          <p class="fr-m-0 fr-text--xs">{{ t('It is important to associate all the datasets used, as this allows to understand the cross-references and improve the visibility of your reuse.') }}</p>
+        </div>
+      </div>
+    </Well>
+    <fieldset class="fr-fieldset min-width-0" aria-labelledby="description-legend">
+      <legend class="fr-fieldset__legend" id="description-legend" v-if="form.datasets.length > 0">
+        <h2 class="subtitle subtitle--uppercase fr-mb-0">
+          {{ t("Associated datasets") }}
+        </h2>
+      </legend>
+      <div class="w-100" ref="el">
+        <div class="fr-col fr-grid-row fr-grid-row--center w-100" v-for="(dataset, index) in form.datasets" :key="dataset.id">
+          <button
+            type="button"
+            class="fr-col-auto fr-btn fr-btn--sm fr-btn--tertiary-no-outline fr-icon-svg fr-my-auto"
+            :title="t('Drag to move this content')"
+          >
+            <img class="fr-mr-2w" :src="draggableIcon" alt="" />
+            {{ t('Drag to move this content') }}
+          </button>
+          <div class="fr-col relative" :style="zIndex(index)">
+            <DatasetCard
+              :dataset="dataset"
+              :dataset-url="dataset.slug"
+              :show-description="false"
+            />
+          </div>
+          <button
+            type="button"
+            class="fr-btn fr-btn--tertiary-no-outline fr-btn--secondary--error fr-icon-close-line border-0 absolute top-3w right-1w"
+            :title="t('Remove the dataset')"
+            @click="removeDataset(index)"
+          >
+            {{ t('Remove the dataset') }}
+          </button>
+        </div>
+      </div>
+      <MultiSelect
```

Comment:
When the multiselect is cleared (with the button), a `change` event is triggered and the first suggestion is added to the list of datasets above.

Reply:
Also, the multiselect keeps its value after the change event whereas the input below resets it. I think both should reset it.